assets/__tests__/rendered_test.js

```diff
@@ -0,0 +1,191 @@
+import { Rendered } from '../js/phoenix_live_view'
+
+describe('Rendered', () => {
```

Comment:
Had to include a lot of diff data for this test file. Don't know if there's a better way to do it but I got it by grabbing data sent over the wire when running [chrismccord/phoenix_live_view_example](https://github.com/chrismccord/phoenix_live_view_example).

Reply:
This is fine for now!  Perhaps some of these can be indented?

Reply:
> Perhaps some of these can be indented?

What do you mean?